.github/labeler.yml

```diff
@@ -25,8 +25,10 @@ tock-libraries:
   - libraries/**/*
 
 WG-OpenTitan:
-  - boards/earlgrey-nexysvideo/**/*
+  - boards/opentitan/**/**/*
+  - boards/opentitan/**/*
```

Comment:
I think you only need the second of these -- the `**` should cover arbitrary depths in the directory structure.